ci-operator/config/openshift/velero/openshift-velero-oadp-1.4.yaml

```diff
@@ -0,0 +1,45 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.22-openshift-4.17
+images:
+- dockerfile_path: Dockerfile.ubi
+  to: velero-oadp-1.4
+- dockerfile_path: Dockerfile-velero-restore-helper.ubi
+  to: velero-restore-helper-oadp-1.4
+promotion:
+  to:
+  - additional_images:
+      velero-oadp-1.4: velero-oadp-1.4
+      velero-restore-helper-oadp-1.4: velero-restore-helper-oadp-1.4
+    name: velero
+    namespace: konveyor
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.13"
```

Comment:
Haven't really looked into if this val is actually used but I imagine some time in the future moving this to 4.16 would make sense.